bitrise/bitrise.go

```diff
@@ -355,6 +359,58 @@ func (artifact BuildArtifact) DownloadArtifact(filepath string) error {
 	return err
 }
 
+// AbortBuild ...
+func (app App) AbortBuild(buildSlug string, abortReason string) error {
+	params := make(map[string]interface{})
+	params["abort_reason"] = abortReason
```

Comment:
The abort reason does not show up on the build that got aborted but I am not sure why.

Is this the correct way to build up the Abort request body?